Add pre-checkpoint and restore with previous (#8819)
containers/podman: docs/source/markdown/podman-container-restore.1.md

Using **--ignore-static-mac** tells Podman to ignore the MAC address if it was
configured with **--mac-address** during container creation.

> #### **--import-previous**

Author: Ok.

> Import a pre-checkpoint tar.gz file if need, which was exported by Podman. The option

Reviewer: ```suggestion
Import a pre-checkpoint tar.gz file which was exported by Podman. This option
```
